packages/react-next/src/common/_constants.scss

```diff
@@ -0,0 +1,12 @@
+$fontWeightLight: 100;
```

Comment:
Let me know if there's a better place to define these values instead of a new file, or if there's another preferred way to define constants like this.